anchor/src/main.rs

```diff
@@ -134,3 +149,126 @@ fn start_anchor(anchor_config: Node, mut environment: Environment) {
         }
     };
 }
+
+fn enable_logging(anchor_config: &Node) -> (Option<WorkerGuard>, Option<Libp2pDiscv5TracingLayer>) {
+    let config = match config::from_cli(anchor_config) {
+        Ok(config) => config,
+        Err(_) => {
+            return (None, None);
+        }
+    };
+
+    let default_logs_dir = if anchor_config.datadir.clone().is_none() {
+        dirs::home_dir()
+            .unwrap_or_else(|| PathBuf::from("."))
+            .join(DEFAULT_ROOT_DIR)
+            .join(
+                config
+                    .ssv_network
+                    .eth2_network
+                    .config
+                    .config_name
+                    .as_deref()
+                    .unwrap_or("custom"),
+            )
+            .join("logs")
+    } else {
+        anchor_config
+            .datadir
+            .clone()
+            .unwrap_or_else(|| PathBuf::from("."))
+            .join("logs")
+    };
```

Comment:
We can simplify this a bit and avoid some clones :)

```suggestion
    let default_logs_dir = if let Some(datadir) = &anchor_config.datadir {
        datadir.join("logs")
    } else {
        dirs::home_dir()
            .unwrap_or_else(|| PathBuf::from("."))
            .join(DEFAULT_ROOT_DIR)
            .join(
                config
                    .ssv_network
                    .eth2_network
                    .config
                    .config_name
                    .as_deref()
                    .unwrap_or("custom"),
            )
            .join("logs")
    };
```